tests/e2e/tests/regression/workbench/command-results.e2e.ts

```diff
@@ -117,3 +118,20 @@ test('Big output in workbench is visible in virtualized table', async t => {
     // Verify that all commands scrolled
     await t.expect(lastExpectedItem.visible).ok('Final execution time message not displayed');
 });
+
+test.before(async t => {
+    await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneRedisearch, ossStandaloneRedisearch.databaseName);
+    // Add index and data
+    await t.click(myRedisDatabasePage.workbenchButton);
+}).after(async t => {
+    await t.switchToMainWindow();
+    await deleteStandaloneDatabaseApi(ossStandaloneRedisearch);
+})('verify client list plugin shows table', async t => {
+    const command = 'CLIENT LIST';
+    // Send command in workbench to view client list
+    await workbenchPage.sendCommandInWorkbench(command);
+    await t.expect(workbenchPage.typeSelectedClientsList.visible).ok('client list view button is not visible');
+    await t.switchToIframe(workbenchPage.iframe);
+    // Verify that I can see the client List visualization available for all users.
+    await workBenchActions.verifyClientListColumnsAreVisible();
+});
```

Comment:
you can automate this scenarios as well:
"Verify that user can see the Client List visualization available for all users."
by checking the number of rows in table and comparing with the number of rows in text view
and also this scenario can be automated
Verify that user have the following error when there is no permission to run the CLIENT LIST: "NOPERM this user has no permissions to run the 'CLIENT LIST' command or its subcommand"